src/actions/index.js

```diff
@@ -81,6 +81,11 @@ export const skillBuff = (message) => {
   }
 }
 
+export const removeBuff = buff => ({
```

Comment:
Instead of mutating the original buffs object (a redux nicht nicht), every state mutation needs its own handler case in a reducer